ecl/hql/hqlutil.cpp

```diff
@@ -5709,7 +5711,8 @@ IHqlExpression * extractCppBodyAttrs(unsigned lenBuffer, const char * buffer)
                 }
             }
         }
-        prev = buffer[i];
+        if (!ignore)
+            prev = next;
```

Comment:
Very well spotted.

I think the previous code also has an issue - e.g., /*abc*/*  this will be ignored*/

I'll try again!